docs/src/main/asciidoc/datasource.adoc

```diff
@@ -3,62 +3,61 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Datasources
+[id="datasources"]
+= Configure datasources in Quarkus
 include::_attributes.adoc[]
-:categories: data
-:summary: With Quarkus, you can easily configure a datasource, or several if need be.
+:categories: data,getting-started
```

Comment:
Should this topic be found when filtering for Reactive content on the landing page? If so, add reactive here:

```suggestion
:categories: data,getting-started,reactive
```